docs/modules.md

```diff
@@ -34,18 +38,125 @@
 
 ### Type aliases
 
+- [ConditionalTransferTransaction](modules.md#conditionaltransfertransaction)
+- [DepositTransaction](modules.md#deposittransaction)
+- [ExcludeRequest](modules.md#excluderequest)
+- [FalseFullWithdrawalTransaction](modules.md#falsefullwithdrawaltransaction)
+- [FullWithdrawalTransaction](modules.md#fullwithdrawaltransaction)
 - [GatewayRequest](modules.md#gatewayrequest)
+- [MintTransaction](modules.md#minttransaction)
+- [MultiTransactionTransaction](modules.md#multitransactiontransaction)
+- [NumericSequence](modules.md#numericsequence)
+- [SettlementTransaction](modules.md#settlementtransaction)
 - [StarkExCertsConfig](modules.md#starkexcertsconfig)
+- [TransferTransaction](modules.md#transfertransaction)
+- [WithdrawalTransaction](modules.md#withdrawaltransaction)
```

Comment:
I know those docs are generated automatically, but maybe to struct the code some how that all `transactions` will be in same place without other types?

Reply:
@danielgluskin-starkware  Agree, let's do it.
Please checkout this branch and add it.
Thanks